batch/Dockerfile

```diff
@@ -24,7 +24,7 @@ ENV OPENMRS_PASSWORD="Admin123"
 ENV SINK_PATH=""
 ENV SINK_USERNAME=""
 ENV SINK_PASSWORD=""
-ENV SEARCH_LIST="Patient,Encounter,Observation"
+ENV RESOURCE_LIST="Patient,Encounter,Observation"
```

Comment:
It would be nice to change docker-compose files e.g here  https://github.com/GoogleCloudPlatform/openmrs-fhir-analytics/blob/353cb17d30ee0e74816a58d1d56eddfecb8c77a3/docker/docker-compose.yaml#L32